onedrive/src/main/java/ch/cyberduck/core/onedrive/features/GraphAttributesFinderFeature.java

```diff
@@ -89,6 +92,13 @@ private DriveItem.Metadata toMetadata(final Path file, final DriveItem item) thr
     public PathAttributes toAttributes(final DriveItem.Metadata metadata) {
         final PathAttributes attributes = new PathAttributes();
         attributes.setETag(metadata.getETag());
+        final File file = metadata.getFile();
+        if(file != null) {
+            final Hashes hashes = file.getHashes();
+            if(hashes != null) {
+                attributes.setChecksum(Checksum.parse(hashes.getQuickXorHash()));
```

Comment:
This relies on quickXorHash always being 20 Bytes, and is misrepresented as SHA1.

Reply:
You are right. Full implementation requires us to implement `quickXorHash` as a `ChecksumCompute` feature.